Section_I/law4.tex

```diff
@@ -148,6 +148,7 @@
 
 Robots may communicate only via the (wireless) network provided by the organizers, which must support the referee box. The total bandwidth of each robot belonging to one team may not exceed 1 Mbit/s. In the virtual competition, teams will not be able to monitor the robot communication and receive debug messages during an ongoing simulation. In the physical competition, the robots must not rely on the quality of the wireless network. They
 must be able to play if the network is of low quality. During a physical competition, only robots are allowed to communicate by WLAN. Any other computers of team members are only allowed to communicate by tethered LAN. No other wireless communication is allowed onsite. All other wireless hardware must be deactivated. A team may be disqualified if one of the team members violates this rule.
+\added{To ensure that no devices are accidentally connected to the wireless network, the password of the network is changed after every game. The passwords will be provided by the Technical Committee in time for the games.}
```

Comment:
I think we should specify what "in time" means. Given that games are sometimes played back to back, I think this cannot be more than 15 minutes before a game.